src/router/task.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express')
+const Task = require('../models/task')
+const router = new express.Router()
+
+router.post('/', async (req, res) => {
```

Comment:
What happens with url when you add another entity? Check [REST-API-Design-Rulebook](https://pepa.holla.cz/wp-content/uploads/2016/01/REST-API-Design-Rulebook.pdf)